lib/src/initializer.dart

```diff
@@ -67,6 +67,12 @@ $tool=$dateStamp,$toolsMessageVersion
 ''');
   }
 
+  /// Creates that log file that will store the record formatted
+  /// events locally on the user's machine
+  void createLogFile({required File logFile}) {
+    logFile.createSync(recursive: true);
```

Comment:
good point, there are several places where we do synchronous operations, I have created an issue for this so we can do them all at once instead of piece by piece

https://github.com/eliasyishak/dash_analytics/issues/26